### ISSUE TITLE
fsaverage read only on shared system 

### ISSUE THREAD

**Issue by agramfort** [mne-tools/mne-biomag-group-demo#27]
when running on the shared computer for which freesurfer was installed with root access these lines fail:

```
# now we do something special for fsaverage
fsaverage_src = op.join(subjects_dir, 'fsaverage', 'bem')
if not op.isdir(fsaverage_src):
    os.mkdir(fsaverage_src)
fsaverage_src = op.join(fsaverage_src, 'fsaverage-5-src.fif')
if not op.isfile(fsaverage_src):
    print('Setting up source space for fsaverage')
    src = mne.setup_source_space('fsaverage', 'ico5',
                                 subjects_dir=subjects_dir)
    for s in src:
        assert np.array_equal(s['vertno'], np.arange(10242))
    mne.write_source_spaces(fsaverage_src, src)
```

basically we should fsaverage from freesurfer app folder to be able to write in it.

I am taking a look now.

**Comment by agramfort** [comment]
fixed in 855bffd1aa232a34960052f1da80c73801c666ce

**Comment by jasmainak** [comment]
Thanks for the fix!
